resources_pages/install_ds_stack_windows.md

```diff
@@ -491,7 +498,7 @@ To improve the experience of using R in JupyterLab,
 we will add an extension that allows us to setup keyboard shortcuts for inserting text
 (thanks to former MDS student Ryan Homer for developing this extension!).
 By default,
-it creates shortcuts for inserting two of the most common R operators: `<-` and `%>%`.
+it creates shortcuts for inserting two of the most common R operators: `<-` and `|>`.
```

Comment:
Could you make changes like this in all three instructions (mac, win, linux?), so that they don't go out of sync with eachother and we forget one of them.